lang/main.json

```diff
@@ -773,6 +773,7 @@
         "enterRoomTitle": "Start a new meeting",
         "go": "GO",
         "join": "JOIN",
+        "numbers": "Numbers",
```

Comment:
"Info" ?

Reply:
Tried the 2 lines rendering again, didn't work, so I fell back to "Dial-in". What do you think?

Reply:
Sounds good!

Reply:
It may look worse on other languages though.

Reply:
Ok, wen't back to ```Info``` :D